fairer-iOS/fairer-iOS/Screens/CreateHouseWork/CreateHouseWorkViewController.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  CreateHouseWorkViewController.swift
+//  fairer-iOS
+//
+//  Created by LeeSungHo on 2022/09/20.
+//
+
+import UIKit
+
+import SnapKit
+
+final class CreateHouseWorkViewController: BaseViewController {
+    
+    // MARK: - property
+    
+    private let spaceCollectionView = SelectHouseWorkSpaceCollectionView()
+    
+    // MARK: - life cycle
+    override func viewDidLoad() {
```

Comment:
p5;

```suggestion
    // MARK: - life cycle
    
    override func viewDidLoad() {
```

한 칸 가능한가요...?? ^_______^